packaging/windows/WiXTools/project.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilationOptions": {
+      "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "WiX": "3.10.0.2103-pre1"
```

Comment:
I tested that on uninstall the environment variables are in fact removed.  In case of %path% the path to dotnet is also removed.

Here is the documentation for the [Permanent](http://wixtoolset.org/documentation/manual/v3/xsd/wix/environment.html) attribute which controls it
>Permanent (yesorno) Specifies that the environment variable should not be removed on uninstall.